crates/prover/src/components/opcodes/store_lower_than_fp_imm.rs

```diff
@@ -0,0 +1,766 @@
+//! This component is used to prove the StoreLowerThanFpImm opcode.
+//! [fp + dst_off] = [fp + src_off] < imm
+//!
+//! Math from https://github.com/starkware-libs/cairo-lang/blob/master/src/starkware/cairo/common/math.cairo#L161-L228
+//!
+//! # Columns
+//!
+//! - enabler
+//! - pc
+//! - fp
+//! - clock
+//! - inst_prev_clock
+//! - src_off
+//! - imm
+//! - dst_off
+//! - src_val
+//! - src_prev_clock
+//! - dst_prev_val
+//! - dst_prev_clock
+//! - a (where a is supposed to be <= b)
+//! - b
+//! - diff_inv
+//! - keep_0
+//! - keep_1
+//! - keep_2
+//! - arc_short_lo
+//! - arc_short_hi
+//! - arc_long_lo
+//! - arc_long_hi
+//! - invert
+//! - is_lt
+//!
+//! # Constraints
+//!
+//! PRIME_OVER_3_HIGH = ceil(PRIME / 3 / 2 ** 16)
+//! PRIME_OVER_2_HIGH = ceil(PRIME / 2 / 2 ** 16)
+//! arc_short = arc_short_lo + arc_short_hi * PRIME_OVER_3_HIGH
+//! arc_long = arc_long_lo + arc_long_hi * PRIME_OVER_2_HIGH
+//! arc_sum = arc_short + arc_long
+//! arc_prod = arc_short * arc_long
+//!
+//! * enabler is a bool
+//!   * `enabler * (1 - enabler)`
+//! * keep_{i} is a bool
+//!   * `keep_0 * (1 - keep_0)`
+//!   * `keep_1 * (1 - keep_1)`
+//!   * `keep_2 * (1 - keep_2)`
+//! * two of keep_0, keep_1, keep_2 are equal to 1
+//!   * `enabler * (keep_0 + keep_1 + keep_2 - 2)`
+//! * diff_inv is the inverse of diff or diff is 0
```

Comment:
```suggestion
//! * diff = a - b
//! * diff_inv is the inverse of diff or diff is 0
//!   * `- diff * (diff_inv * diff - 1)`
//! * diff_inv is the inverse of diff or diff_inv is 0
//!   * `- diff_inv * (diff_inv * diff - 1)`
//! * is_lt is 1 if a < b, 0 otherwise
//!   * `- is_lt - (1 - invert) * diff_inv * (a - b)`
```